composer.json

```diff
@@ -11,7 +11,8 @@
     ],
     "require": {
         "php": "~5.6|~7.0",
-        "ezsystems/ezpublish-kernel": "^6.0|^7.0"
+        "ezsystems/ezpublish-kernel": "^6.0|^7.0",
+        "netgen/ezplatform-search-extra": "^1.0-dev"
```

Comment:
I think the standard notation is `^1.0@dev`

Reply:
Yup, amended the commit.